.werf/defines/image-mountpoints.tmpl

```diff
@@ -0,0 +1,23 @@
+# add mount points from file mount_points.yaml in the build dir
+
+{{- define "image mount points" }}
+  {{- $mountPoints := ($.Files.Get (printf "images/%s/mount-points.yaml" $.ImageName) | fromYaml) }}
+  {{- range $v := $mountPoints.dirs }}
+- add: /tools/mounts/mountdir
```

Comment:
Module is embedded in cse, so git paths should be prefixed with ModuleDir variable:

```suggestion
- add: {{ .ModuleDir }}/tools/mounts/mountdir
```

Reply:
`ModuleDir` not needed. cse version has its own copy of "image mount points" template with correct calculations for paths.